WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsHelper.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import WordPressReader
+
+/// A collection of utilities for managing rendering for comments.
+@MainActor
+@objc class ReaderCommentsHelper: NSObject {
+    private var contentHeights: [TaggedManagedObjectID<Comment>: CGFloat] = [:]
+    private let renderers = NSCache<Comment, WebCommentContentRenderer>()
+
+    override init() {
+        renderers.countLimit = 30
+    }
+
+    func getRenderer(for comment: Comment) -> WebCommentContentRenderer {
+        if let renderer = renderers.object(forKey: comment) {
```

Comment:
I'm not sure it's a good idea to reuse web view across different cells this way, but it helps with cell reload. `UITableView` does little to nothing to ensure the same cells are used for the same comments on reload.